packages/docusaurus/src/webpack/plugins/LogPlugin.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import WebpackBar from 'webpackbar';
+import {Compiler} from 'webpack';
+
+/**
+ * react-dev-utils' formatWebpackMessages but for webpack v5 - also its much shorter
+ */
+export function formatWebpackMessages5(stats?: {
```

Comment:
👀 not using `react-dev-utils/formatWebpackMessages`, what are the possible side effects?

`react-dev-utils` does not yet support webpack 5: https://github.com/facebook/create-react-app/issues/9994

Reply:
It literally throws an error

Reply:
sure, but maybe we could provide our own compat layer somehow.

I don't know yet what this lib does so it's hard to say if we really need it 😅

Reply:
For now let's keep using `react-dev-utils/formatWebpackMessages`: I provided a compat layer